tools/autograd/gen_variable_type.py

```diff
@@ -1419,6 +1422,7 @@ def emit_fw_derivatives() -> List[str]:
                             out_arg=res[0], is_inplace=is_inplace_str
                         )
                     )
+                    requires_fw_grad += f" && ({derivative.var_names[0]}.defined())"
```

Comment:
Don't we already check above that this is defined in https://github.com/pytorch/pytorch/blob/1afb804f26c960aec32aed7c587dcb483be8c580/torch/csrc/autograd/functions/utils.h#L100 ?

Reply:
Yeah that is only applied to the inputs of the jvp though